motoman_mpx1950_support/package.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>motoman_mpx1950_support</name>
+  <version>0.1.0</version>
+  <description>
+    <p>ROS-Industrial support for the Motoman MPX1950 (and variants).</p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for Motoman MPX1950 manipulators. This currently includes the base model
+      which also supports the DT and XP variants.
```

Comment:
Is this correct? Are there `DT` and `XP` variants, or is this copy-pasta?

Reply:
No, I removed it.